integration/spark-common-test/pom.xml

```diff
@@ -160,6 +165,35 @@
       <artifactId>jmockit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.alluxio</groupId>
+      <artifactId>alluxio-core-client-hdfs</artifactId>
+      <version>1.8.1</version>
```

Comment:
consider to add a project configuration for this which is friendly to upgradation.

Reply:
This is only for test, not public. It's better to keep in test module and avoid increase the risk for other module.

Reply:
then you can add the project configuration only in this pom

Reply:
still hang here?

Reply:
What your mean? Do you mean add alluxio version in properties of parent pom and invoke it in here?